lib/prompts/promptUtils.ts

```diff
@@ -42,3 +42,22 @@ export async function listPrompt(
   ]);
   return choice;
 }
+
+export async function inputPrompt(
+  message: string,
+  {
+    when,
+  }: {
+    when?: boolean | (() => boolean);
+  } = {}
+): Promise<string> {
+  const { input } = await promptUser([
+    {
+      name: 'input',
+      type: 'input',
+      message,
+      when,
+    },
+  ]);
+  return input;
+}
```

Comment:
we really just need to upgrade to the new inquirer haha

Reply:
Yeah haha.  The current version we are on its pretty unintuitive